test/kt/CMakeLists.txt

```diff
@@ -162,12 +165,18 @@ function(_generate_gpu_scan_tests)
     foreach (_data_per_work_item ${_data_per_work_item_all})
         foreach (_work_group_size ${_work_group_size_all})
             foreach (_type ${_type_all})
-                _generate_gpu_scan_test(${_data_per_work_item} ${_work_group_size} ${_type})
+                # ~1 test: (6/1000) * (6 * 5 * 6)
+                _generate_gpu_scan_test(${_data_per_work_item} ${_work_group_size} ${_type} 6)
             endforeach()
         endforeach()
     endforeach()
 endfunction()
 
 if (ONEDPL_TEST_ENABLE_KT_SYCL)
     _generate_gpu_scan_tests()
+
+    # Pin some cases to track them
+    _generate_gpu_scan_test("8" "512" "uint32_t" 1000)
+    _generate_gpu_scan_test("4" "256" "float" 1000)
+    _generate_gpu_scan_test("16" "64" "int64_t" 1000)
```

Comment:
@adamfidel, could you suggest the most important cases?

Reply:
I think these three are a good representative sample for the scan kernel template.